lib/src/protocol/v1/serialization.cc

```diff
@@ -9,7 +9,7 @@
 namespace PCPClient {
 namespace v1 {
 
-#ifdef BOOST_ENDIAN_LITTLE_BYTE
+#if BOOST_ENDIAN_LITTLE_BYTE
```

Comment:
This looks wrong. https://github.com/puppetlabs/cpp-pcp-client/commit/741ee6897088607737f28acce2d83592e65d7a00

Reply:
yet, April 12: https://github.com/puppetlabs/cpp-pcp-client/commit/36ec2eba1ee6c0edeaaaaad9836041f84a02e463#diff-03a0c044596526965eba33d65a5657dbR12

Reply:
🤷

Reply:
Yes, it does look wrong. Not sure how `#if...#endif` even compiles.

Reply:
Updated to go with what was in master. Should that change in 1.6.x too?

Reply:
Yes. I missed that on my review for the PR that was just merged. https://github.com/puppetlabs/cpp-pcp-client/pull/254 my bad

Reply:
#256